web-www/public/_redirects

```diff
@@ -0,0 +1,13 @@
+/nct https://www.notion.so/regennetwork/A-Guide-to-Nature-Carbon-Ton-NCT-8204ea9d20d0436281f49b8fd1b3fbd2
```

Comment:
these were already defined in next.config.js but it's not handled well by netlify: https://docs.netlify.com/frameworks/next-js/runtime-v4/redirects-and-rewrites/#when-to-use-netlify-redirects-or-rewrites